manifests/init.pp

```diff
@@ -316,6 +323,7 @@
   ~> class { '::foreman::install': }
   ~> class { '::foreman::config': }
   ~> class { '::foreman::database': }
+  ~> class { '::foreman::tasks_service': }
```

Comment:
I think that if you do this inside the tasks class with ```contain ::foreman::tasks_service``` then you can just notify the service class and plugins automatically restart the tasks service as well. I think that's a good thing in general even if it's not always needed, it can save some headache. Especially if more plugins start to use background jobs. It will also mean you can remove the inclusion from plugins that only need the tasks service and not bother with the notification.